<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scitula</class>
 <widget class="QDialog" name="Scitula">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPlainTextEdit" name="EncryptedplainTextEdit">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Encrypted message</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTextBrowser" name="DecryptedtextBrowser">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Decrypted message</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPlainTextEdit" name="MessageplainTextEdit">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Open message</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Key (amount of column):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="KeyspinBox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="EncryptpushButton">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Encrypt</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="DecryptpushButton">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Decrypt</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="2">
    <widget class="QPushButton" name="ExitpushButton">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
